packages/wells/package.json

```diff
@@ -9,7 +9,7 @@
   "version": "0.8.0",
   "scripts": {
     "clean": "rm -rf dist/ docs/ codeSnippets/",
-    "test": "jest --config=../../jest.config.js --testPathPattern=/wells/",
+    "test": "jest --config=../../jest.config.js --testPathPattern=/wells/src",
```

Comment:
when I used yalc to install the package locally, it generated its own test files, and they were being run when I was using `yarn test` so have scoped the test path a little closer to stop this